apps/mark-scan/backend/src/server.test.ts

```diff
@@ -1,25 +1,102 @@
-import { fakeLogger } from '@votingworks/logging';
+import { LogEventId, fakeLogger } from '@votingworks/logging';
 import tmp from 'tmp';
 import { buildMockInsertedSmartCardAuth } from '@votingworks/auth';
+import {
+  BooleanEnvironmentVariableName,
+  getFeatureFlagMock,
+} from '@votingworks/utils';
+import { MockPaperHandlerDriver } from '@votingworks/custom-paper-handler';
+// import { sleep } from '@votingworks/basics';
```

Comment:
Cleaned up in https://github.com/votingworks/vxsuite/pull/4628/commits/1fed52f30b1b41daae362e59af77ba38c864bcfc